.github/workflows/ci.yml

```diff
@@ -172,7 +172,10 @@ jobs:
           cd ${GITHUB_WORKSPACE}/build
           echo Run ctest -V -L DFLY
           GLOG_alsologtostderr=1 GLOG_vmodule=rdb_load=1,rdb_save=1,snapshot=1 \
-          FLAGS_fiber_safety_margin=4096 FLAGS_list_experimental_v2=true timeout 20m ctest -V -L DFLY
+          FLAGS_fiber_safety_margin=4096 FLAGS_list_experimental_v2=true timeout 20m ctest -V -L DFLY -E allocation_tracker
+
+          # Run allocation tracker test separately. It generates a TON of logs
```

Comment:
Allocation tracker test generates a gazillion number of logs. I silenced it that way by excluding the test from running with alsologtostderr flag.

Reply:
Example https://github.com/dragonflydb/dragonfly/actions/runs/13011160524/job/36288993157 :scream: